Makefile

```diff
@@ -16,3 +16,9 @@ run:
 test: ruff pylint
 	export PYTHONPATH=src/main/py; \
 	python3 -m unittest discover --pattern "*_test.py" --start-directory src/test/py --top-level-directory .
+
+clean:
```

Comment:
Please use target name `clean-env` to make it easier to roll up various `clean` targets into an overall one.